packages/core/scripts/build/copyFiles.ts

```diff
@@ -27,7 +27,7 @@ interface CopyTask {
 const tasks: CopyTask[] = [
     { target: path.join('src', 'templates') },
     { target: path.join('src', 'test', 'shared', 'cloudformation', 'yaml') },
-    { target: path.join('src', 'test', 'codewhisperer', 'service', 'resources') },
```

Comment:
Do we need to delete this folder?

Reply:
Its not used anymore. So yes.

Reply:
then can we delete it in this PR before its forgotten about? Looks like you eliminated the last use case.